migrations/migrations.sql

```diff
@@ -5053,6 +5053,10 @@ do $$
       alter table "accounts" drop column "lastHubspotSyncAt";
     end if;
 
+    if not exists(select 1 from information_schema.columns where table_name = 'systemSettings' and column_name = 'maxDiskUsagePercentage') then
```

Comment:
We add `systemSettings` changes in [here](https://github.com/Shippable/admiral/blob/master/common/scripts/configs/system_settings.sql.template).